code/game/machinery/doors/airlock_electronics.dm

```diff
@@ -66,13 +69,17 @@
 		if("clear_all")
 			accesses = list()
 			one_access = 0
+			maintenance = TRUE
 			. = TRUE
 		if("grant_all")
 			accesses = get_all_accesses()
 			. = TRUE
 		if("one_access")
 			one_access = !one_access
 			. = TRUE
+		if("maintenance")
```

Comment:
```suggestion
		if("toggle_maint_emergency")
```
writing an action name as the same as a var name is not a good standard.
I know you copypasta'ed `one_access`, but what I am saying is better to read.